src/group_theory/group_action/fixing_subgroup.lean

```diff
@@ -0,0 +1,201 @@
+/-
+Copyright (c) 2022 Antoine Chambert-Loir. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir
+-/
+
+import group_theory.subgroup.basic
+import group_theory.group_action.basic
+import order.order_dual
+
+/-!
+
+# Fixing submonoid, fixing subgroup of an action
+
+In the presence of of an action of a monoid or a group,
+this file defines the fixing submonoid or the fixing subgroup,
+and relates it to the set of fixed points via a Galois connection.
+
+## Main definitions
+
+* `fixing_submonoid M s` :
+in the presence of `mul_action M α` (with `monoid M`)
+it is the `submonoid M` consisting of elements which fix `s : set α` pointwise.
```

Comment:
This line and the one above it should be indented to align with the first line of the bullet point

(And throughout)

Reply:
Done